pkg/platform/yaml/yaml.go

```diff
@@ -259,9 +259,14 @@ func (ys *Yaml) ReadTestcase(path string, lastSeenId platform.KindSpecifier, opt
 		path = ys.TcsPath
 	}
 
+	path, err := util.ValidatePath(path)
+	if err != nil {
+		return nil, err
```

Comment:
instead of using `path` ,`testPath` would be a better assigned name as to avoid confusion + maintain consistency with other functions across repo.